src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -91,8 +91,8 @@ internal static bool DoSslHandshake(SafeSslHandle context, IntPtr recvPtr, int r
                     int errorCode = sendCount;
                     Marshal.FreeHGlobal(sendPtr);
                     sendPtr = IntPtr.Zero;
-                    sendCount = 0;                  
-                    throw CreateSslException(context, "Read Bio failed: ", errorCode);                   
+                    sendCount = 0;
+                    Debug.Fail("SSL Read BIO failed with error:" + errorCode);
```

Comment:
Actually it should throw as we won't be throwing inside of BioRead & Write. Will update.